packages/core/integration-tests/test/babel.js

```diff
@@ -208,7 +207,7 @@ describe('babel', function() {
     assert(file.includes('function Bar'));
   });
 
-  it('should compile node_modules with browserslist to app target', async function() {
+  it.skip('should compile node_modules with browserslist to app target', async function() {
```

Comment:
Shouldn't this test still be run, seems like if you would target IE11 you'd expect a bundler to compile node_modules as well? At minimum we should be able to opt-in to this behaviour in some way

Most bundlers don't but that does not really seem like a good idea to me as it will cause builds to fail on these platforms unexpectedly.